scripts/lb-wrapper

```diff
@@ -26,11 +20,13 @@ log() {
     echo "$(date +'%Y-%m-%d %H:%M:%S') - [$level] $message" | tee -a ${LOG_FILE}
 }
 
-if [ $# -eq 0 ]; then
-    log "Error" "No arguments provided. Please provide a URL to download."
+if [ $# -lt 2 ]; then
+    log "Error" "Insufficient arguments provided. Please provide a command (tubeadd/dl) and a URL."
     exit 1
 fi
 
+command_type=$1
```

Comment:
Personally I'd prefer these 2 right on the very top of the file, as the very first constants declared...

```
XKLB_COMMAND="$1"
URL="$2"
```

...to make the bash script more readable 🙏

Reply:
Or if necessary... ?

```
XKLB_INTERNAL_CMD="$1"
URL="$2"
```

And maybe preserve the original `XKLB_FULL_CMD` variable name to reduce confusion?

Reply:
> XKLB_INTERNAL_CMD="$1"
URL="$2"

Using "$1" and add append the URL at the end makes the command ugly and difficult to grasp

> And maybe preserve the original XKLB_FULL_CMD variable name alongside, to reduce confusion?

We can't per the design of separating them to decouple metadata fetching and video downloading

Reply:
It sounds like you're not understanding my suggestions.

I'll try to reach you privately.

Reply:
Thanks for explaining. Done.